Add instruction for install Feast on IKS and OpenShift using Kustomize (#1534)
feast-dev/feast: docs/feast-on-kubernetes/getting-started/install-feast/ibm-cloud-iks-with-kustomize.md

   ```text
    ibmc-block-gold (default)   ibm.io/ibmc-block   65s
   ```

> ### Security Context Constraint Setup

Reviewer: ```suggestion
### Security Context Constraint Setup (OpenShift only)
```

> By default, in OpenShift, all pods or containers will use the [Restricted SCC](https://docs.openshift.com/container-platform/4.6/authentication/managing-security-context-constraints.html) which prevents some pods to run as certain UID and causes installation to fail. In order to grant additional permissions, beyond those acquired with the restricted SCC, we need to use a different SCC. Since pods are created under the default or specified service accounts under the namespace, adding them to the `anyuid` SCC will resolve the failure. To do so, execute the following:

Reviewer 2: ```suggestion
By default, in OpenShift, all pods or containers will use the [Restricted SCC](https://docs.openshift.com/container-platform/4.6/authentication/managing-security-context-constraints.html) which limits the UIDs pods can run with, causing the Feast installation to fail. To overcome this, you can allow Feast pods to run with any UID by executing the following:
```

```text
oc adm policy add-scc-to-user anyuid -z default,kf-feast-kafka
```
## 3. Installation

Install Feast using kustomize. The pods may take a few minutes to initialize.

```bash
git clone https://github.com/kubeflow/manifests
cd manifests/contrib/feast/
kustomize build feast/base | kubectl apply -n feast -f -
```
### Optional: Enable Feast Jupyter and Kafka

> You may optionally enable the Feast Jupyter which contains code examples to demonstrate Feast. Some examples require Kafka to stream real time features to the Feast online serving. To enable, edit the following properties in the `values.yaml` under the `manifests/contrib/feast` folder:

Reviewer 2: add "component" after Jupyter.

```
kafka.enabled: true
feast-jupyter.enabled: true
```

Then regenerate the resource manifests and deploy:
```
make feast/base
kustomize build feast/base | kubectl apply -n feast -f -
```

> ## 4. Use Feast Jupyter to connect to Feast

Reviewer: ```suggestion
## 4. Use Feast Jupyter Notebook Server to connect to Feast
```
